template/pagetype-doc.html.jinja

```diff
@@ -26,6 +26,12 @@
       <p class="mb-0">{% trans %}We are making an effort to offer the XRP Ledger Dev Portal in a variety of languages, but not all pages are available in all languages. If you'd like to help, <a href="https://github.com/XRPLF/xrpl-dev-portal/blob/master/CONTRIBUTING.md">please contribute!</a>{% endtrans %}</p>
     </div><!--/.devportal-callout-->
     {% endif %}
+    {% if target.lang != "en" and currentpage.outdated_translation %}
+    {# Add a "This translation is not up-to-date." banner. #}
+    <div class="mb-5 devportal-callout note"><strong>{% trans %}This page has the latest version, but it is not provided in this language.{% endtrans %}</strong>
```

Comment:
This phrase doesn't carry the right nuance in English. I think the following phrasing is clearer:

```suggestion
    <div class="mb-5 devportal-callout note"><strong>{% trans %}This translated page is not updated to match the latest changes in the English version.{% endtrans %}</strong>
```

Of course, this text will only ever be displayed in translation, so it's a little strange to edit here, but I think making the intent clearer in English will help potential future translations.